docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -333,6 +333,7 @@ Return low-level information on the container `id`
 			"NetworkDisabled": false,
 			"OnBuild": null,
 			"OpenStdin": false,
+            "PortSpecs": null,
```

Comment:
@runcom looks like the lines above your change are using tabs :'(

In this case, perhaps you can change it to use a tab as well, then we can fix it up later to convert all tabs to spaces in a separate PR.

You can check that by selecting/highlighting the whitespace with your mouse and dragging (well, something like that)